cumulusci/core/exceptions.py

```diff
@@ -2,13 +2,18 @@
 class CumulusCIException(Exception):
     pass
 
+class CumulusCIUsageError(CumulusCIException):
+    pass
+
+class CumulusCIFailure(CumulusCIException):
+    pass
```

Comment:
Can we give these docstrings explaining that they're meant to be abstract exceptions and when to use them?